cmd/virt-handler/virt-handler.go

```diff
@@ -98,7 +98,7 @@ const (
 	podIpAddress = ""
 
 	// This value is derived from default MaxPods in Kubelet Config
```

Comment:
Following the change to 1000, the above comment is no longer relevant.

Reply:
Updated, thanks!